spec/controllers/sign_up/registrations_controller_spec.rb

```diff
@@ -23,7 +23,9 @@
 
     it 'gracefully handles invalid formats' do
       @request.env['HTTP_ACCEPT'] = "nessus=bad_bad_value'"
-      get :new
+
+      expect { get :new }.
+        to raise_error(Mime::Type::InvalidMimeType)
 
       expect(response.status).to eq(200)
```

Comment:
how does this return a 200 if we render the 406 page?

Reply:
Hm, I misunderstood how this works. Since it's in the test environment, it always raises and doesn't render the 406 page as I expected.

If I set `config.consider_all_requests_local = false` it renders like it would in prod, but I can't find a way to test that this gets rendered appropriately in test.

Reply:
Probably needs to be a request spec ---  I would remove the `response.status` then and just let it assume that the error being raised is appropriate